pkg/utils/metrics/actions.go

```diff
@@ -0,0 +1,21 @@
+package metrics
+
+// Action labels for different pod/container/image operations.
+const (
+	ActionCreate    = "create"
```

Comment:
How about changing all this const variables' naming to be ActionXXXLable?
I think this will be a little bit more readable.